server/tests/participant-status.test.js

```diff
@@ -267,6 +280,34 @@ describe('Test Participant status data model and service', () => {
     // Check result for participant
     const filteredOpen = resultOpen.data.filter((p) => p.id === participant.id);
     expect(filteredOpen.length).toBe(1);
+
+    // Now check duel statuses
+    await setParticipantStatus(
+      emp1,
+      participant.id,
+      'prospecting',
+      {
+        site: site2.siteId,
+      },
+      {
+        sites: [site1.siteId, site2.siteId],
+      }
+    );
+
+    // Get statuses with emp1 for duel statuses for mult org employee
+    const resultDuel = await getParticipants(
```

Comment:
Not very important, but the word is "dual" not "duel"- may cause some confusion. :)

Reply:
Thanks